app/views/events/hbx_enrollment/_policy.xml.haml

```diff
@@ -20,12 +20,14 @@
             %dba= employer.dba
           %fein= employer.fein
         %total_employer_responsible_amount=(hbx_enrollment.total_employer_contribution).round(2)
+        %composite_rating_tier_name= "urn:openhbx:terms:v1:composite_rating_tier##{hbx_enrollment.enrollment_composite_rating_tier}" if hbx_enrollment.benefit_group.plan_option_kind == "single_plan" # for only composite rated benefit group, if benefit group has single plan then it is composite rated benefit group
```

Comment:
Actually in the new code, the method is benefit group identifier is 'sole source'.  Also make sure you are checking for dental - as dental should not have this.